lib/mobility/backends/table.rb

```diff
@@ -85,7 +85,8 @@ module Table
       # @!group Backend Accessors
       # @!macro backend_reader
       def read(locale, **options)
-        translation_for(locale, **options).send(attribute)
+        translation = translations.in_locale(locale)
+        translation.send(attribute) if translation
```

Comment:
Hmm I think this is what the `presence` plugin is for. Also you have thrown away the `**options` here which will be a problem.